github-config.yaml

```diff
@@ -357,3 +357,15 @@ orgs:
         privacy: closed
         repos:
           workshop-apps: admin
+      DevSecOps:
+        description: A team that works on the Dev Sec operations.
+        maintainers:

```

Comment:
can you assign the users as team members instead please?

Reply:
is there any particular reason for that ? @tumido

Reply:
the only difference between team member and maintainer is that a maintainer can add/remove members to the team via UI. Since we use automation for GH maintenance anyways, there's no need to have maintainers. In case the automation is not available at the moment, org owners can still act as maintainers for any team.

Automation will override any changes done via UI every time it's executed anyways, so there's no point of having so many maintainers at all. Manual changes through UI are expected to be done only in emergencies. It also means there's no temptation to do the changes right away in the UI for people if they don't have those permissions anyways.

Any reason to have all team members as maintainers?

Reply:
I was just aware maintainer and members are the same with the maintainer having additional permission of adding new members.
so thought there was no harm in having all listed as maintainers. 
Thank you for the reasoning